src/Arcus.Templates.AzureFunctions.Http/Startup.cs

```diff
@@ -35,7 +36,7 @@ public override void Configure(IFunctionsHostBuilder builder)
         {
             IConfiguration config = builder.GetContext().Configuration;
 
-            builder.AddHttpCorrelation();
+            builder.AddHttpCorrelation((HttpCorrelationInfoOptions options) => { });
```

Comment:
This change is because the parameterless `AddHttpCorrelation` method is made obsolete.
However, if you want to add http correlation with just the default options, then I think it is a bit sad that you now need to call this method like this.  (With an empty lambda).  IMHO, it pollutes your code a bit.
Therefore, why don't we keep the parameterless method as an overload, and behind the scenes, call  `AddHttpCorrelation( HttpCorrelationOptions options => {} )`?

Reply:
This discussion is outside the scope of this PR, but it caught my attention now that I see this here.

Reply:
Yes, this change is added bc this PR adds 'treat warnings as errors' and since this deprecated call was a warning, it was an error in the build 😄 . I know, I would want to use the default too, but we can't do that until the next Web API major.

Reply:
We could make this a 'no warn' in the 'author' section of the template project file, so that it can still use the empty lambda but doesn't fail on the deprecated call. Maybe that's an idea.

Reply:
Done in https://github.com/arcus-azure/arcus.templates/pull/559/commits/09ba884f7fdef747fc9028a72da1466d0bdcbbc1